### PR TITLE
net-irc/weechat: add zstd USE-flag; fix LICENSE; add 4.1.0

### PR DESCRIPTION
**Pull request by sevz17**
it is optional[1] now

[1]: https://github.com/weechat/weechat/commit/6bd0c63192e7acefbb7538a22aa061912cfc06f7

### CODE REVIEW

File: net-irc/weechat/weechat-9999.ebuild [hunk 1]
```diff
@@ -32,7 +32,7 @@ PLUGINS="+alias +buflist +charset +exec +fifo +fset +logger +relay +scripts +spe
 # dev-lang/php eclass support is lacking, php plugins don't work. bug #705702
 SCRIPT_LANGS="guile lua +perl +python ruby tcl"
 LANGS=" cs de es fr it ja pl ru"
-IUSE="doc enchant man nls selinux test ${SCRIPT_LANGS} ${PLUGINS} ${INTERFACES} ${NETWORKS}"
+IUSE="doc enchant man nls selinux test zstd ${SCRIPT_LANGS} ${PLUGINS} ${INTERFACES} ${NETWORKS}"
```
Comment (zpuskas) [inline net-irc/weechat/weechat-9999.ebuild:35]: This should be `+zstd` as it's enabled by default in upstream, so we should too.

**Review by zpuskas — CHANGES_REQUESTED** [review]

File: net-irc/weechat/weechat-9999.ebuild (hunk 1 above)
Comment (sevz17) [reply net-irc/weechat/weechat-9999.ebuild]: Done

**Review by zpuskas — APPROVED** [review]
LGTM, can be merged.

**Comment by sevz17** [comment]
@thesamesam

**Comment by sevz17** [comment]
v4.1.0 was just released, I'll do the bump in a few hours.

**Pull request closed without merging**